legal-api/src/legal_api/resources/v2/business/business_tasks.py

```diff
@@ -61,6 +61,13 @@ def get_tasks(identifier):
         else:
             rv = []
     else:
+        compliance_warnings = check_compliance(business)
+        if len(compliance_warnings) > 0:
+            rv = []
+            business.compliance_warnings = compliance_warnings
```

Comment:
Is this right? Are we blocking the users from doing anythingelse till they get all the data right?

Reply:
I'm going to confirm with Linda.  Just implemented this way for now.  I know most things will be locked down.

Reply:
asked Linda whether I should return other tasks as well.  waiting for her answer.

Reply:
reply from Linda.

>Just returning the conversion to do is great. SP/GPs don’t have annual reports or other legislated filings that they have to do with us.

Reply:
What happens if they have already filed a conversion filing, and then move to a state that has compliance warnings in the future?

Reply:
Confirmed with Linda.  The logic of displaying the todo whenever a business has incomplete info can remain even if a conversion filing has been filed already.